### PR TITLE
ISSUE-193: Keep workflows open after create bucket

### LINKED ISSUE #193: Keep Workflows Open After Create Bucket
In the workflow tools, when a new bucket is created, rather than closing the workflow tools popo-up can:

- keep the pop-up open
- create a snackbar that states that a bucket was successfully created
- allow for the creation of another bucket (i.e., by clearing the bucket name field and greying out the Create Bucket button)

### PR DESCRIPTION
**Pull request by Ammar-T**
## Details

- Keep modal open after creating new bucket.

Note: You may have noticed snackbar design/colors are off. We can handle those in a separate issue.

Closes #193

### CODE REVIEW

**Review by JoelWiebe — CHANGES_REQUESTED** [review]
Thanks for your quick work on this.

I just noticed two things:

- if I open the workflow tool, create two buckets, then click Close on the second bucket's snackbar, the first bucket's snackbar will open back up again
- if I then keep the workflow tool open and navigate to the Create Workflow tab, then the source list will show: Board1, Bucket1, Board1, Bucket1, Bucket2

Let me know if you are having trouble replicating this or if I can provide further clarification!

**Comment by Ammar-T** [comment]
@JoelWiebe For your first point, currently that behavior is intended, so if a snackbar is opened but another one opens right after. If the new one closes, then the old one will be shown again. Should I change it so we don't keep track of snackbars that were already closed? This would close all previous snackbars if a new one is opened.

Fixed the second issue.

File: src/app/components/create-workflow-modal/create-workflow-modal.component.ts [hunk 1]
```diff
@@ -74,6 +74,12 @@ export class CreateWorkflowModalComponent implements OnInit {
   ngOnInit(): void {
     this.board = this.data.board
     this.tags = this.data.board.tags
+    this.loadBucketsBoards();
+    this.loadWorkflows()
+    this.tagsFormControl.valueChanges.subscribe((value) => this.selectedTags = value)
```
[the remark this answers is not in the document]
Comment (Ammar-T) [reply src/app/components/create-workflow-modal/create-workflow-modal.component.ts]: Yup, good call.

**Comment by JoelWiebe** [comment]
@Ammar-T Thanks! I guess what was unexpected was if I create the first bucket and let the first snackbar fade away, then I create a second bucket and click close on the second bucket then the first snackbar comes back (but I had expected that it had already faded away). In that case, if it already faded away, does it make sense if we forget about it?

**Comment by Ammar-T** [comment]
Oh yeah, that shouldn't be happening. I'll look into it!

**Comment by Ammar-T** [comment]
@JoelWiebe So I wasn't able to completely fix the issue, which takes me to plan B: Avoid the situation 😆 

I've removed the duration for "Created Bucket..." snackbars so they won't fade away and you'll have to manually close them. Although not ideal for the long term, it fixes the issue of previously closed snackbars showing up again. This can also be looked at in the future.

**Comment by Ammar-T** [comment]
I'm merging this now just so we can have it up on the server, if any issues pop up we can do a hotfix.

**Pull request merged**